File: frontend/src/app/models/entity/DetalleRecetaProg.ts
export class DetalleRecetaProg {
    constructor(

        public codmei            ?: string,
        public meindescri        ?: string,
        public dosis             ?: number,
        public formulacion       ?: number,
        public dias              ?: number,
        public cantsoli          ?: number,
        public cantdespachada    ?: number,
        public cantpendiente     ?: number
    ) { }
}

File: backend/Modulos/models/detalleRecetaProg.go
package models

// DetalleRecetaProg is...
type DetalleRecetaProg struct {
	CodMei         string `json:"codmei"`
	MeInDescri     string `json:"meindescri"`
	Dosis          int    `json:"dosis"`
	Formulacion    int    `json:"formulacion"`
	Dias           int    `json:"dias"`
	CantSoli       int    `json:"cantsoli"`
	CantDespachada int    `json:"cantdespachada"`
	CantPendiente  int    `json:"cantpendiente"`
}
